tools/run_tests/run_xds_tests.py

```diff
@@ -1504,22 +1713,41 @@ def __init__(self, compute, alpha_compute, project):
             test_log_filename = os.path.join(log_dir, _SPONGE_LOG_NAME)
             test_log_file = open(test_log_filename, 'w+')
             client_process = None
+
+            if test_case in _TESTS_TO_RUN_MULTIPLE_RPCS:
+                run_all_rpcs = '--rpc="UnaryCall,EmptyCall"'
+            else:
+                run_all_rpcs = '--rpc="UnaryCall"'
+
+            if test_case in _TESTS_TO_SEND_METADATA:
+                send_metadata = '--metadata="EmptyCall:{key}:{value}"'.format(
+                    key=_TEST_METADATA_KEY, value=_TEST_METADATA_VALUE)
+            else:
+                send_metadata = '--metadata=""'
+
             if test_case in _TESTS_TO_FAIL_ON_RPC_FAILURE:
                 wait_for_config_propagation(
                     gcp, instance_group,
                     args.client_cmd.format(server_uri=server_uri,
                                            stats_port=args.stats_port,
                                            qps=args.qps,
-                                           fail_on_failed_rpc=False),
+                                           fail_on_failed_rpc=False,
+                                           run_all_rpcs=run_all_rpcs,
```

Comment:
Why is this argument called `run_all_rpcs`? Shouldn't it just be `rpcs`, since the flag is `--rpcs`?

Reply:
`rpcs` is quite a short name. I was thinking it may conflict with other variables. Same reason `metadata` is named `send_metadata`

Reply:
That makes sense. But run_all_rpcs implies it runs all rpcs, like a boolean flag. How about `rpcs_to_send`?

Reply:
Or `rpc_methods`

Reply:
Renamed to `rpcs_to_send` and `metadata_to_send`